app/src/interfaces/select-color/select-color.vue

```diff
@@ -406,14 +407,16 @@ function useColor() {
 	padding-right: 0px;
 }
 
-.v-input.html-color-select {
-	width: 0;
-	height: 0;
-	visibility: hidden;
-}
+.color-input {
+	:deep(.input) {
+		padding-left: 6px;
+	}
 
-.color-input :deep(.input) {
-	padding-left: 6px;
+	.v-input.html-color-select {
+		width: 0;
+		height: 0;
+		visibility: hidden;
+	}
```

Comment:
Before this, `.v-input.html-color-select`'s `width: 0` is being overridden by `.v-input.full-width`'s `width: 100%`:

![](https://user-images.githubusercontent.com/42867097/224754724-a4b374d7-4179-4468-b84b-54894b3d5fdd.png)

which causes the weird extra spacing to occur.

After putting `.v-input.html-color-select` inside `.color-input`, it now properly has higher CSS specificity:

![](https://user-images.githubusercontent.com/42867097/224755130-8b48ac8d-1091-474f-895c-c6097c745ad3.png)